src/angular-versions.ts

```diff
@@ -232,5 +232,34 @@ export const versions = new Map<string, PackageJsonVersion>([
         typescript: '~4.0.2'
       }
     }
+  ],
+  [
+    '11.0.x',
+    {
+      dependencies: {
+        '@angular/animations': '~11.0.0-rc.1',
```

Comment:
Could we use `">=11.0.0-rc.1 || >=11.0.0 <11.1.0"`? for all of these and a similar version range for the devkit dependencies?

Reply:
I just verified locally that this version range works. Currently, it resolves to `11.0.0-rc.1` using Yarn CLI.

Reply:
Great! I will update it.